modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java

```diff
@@ -319,8 +329,20 @@ private Map<String, EntityField> _getStringEntityFieldsMap(
 		return entityFieldsMap;
 	}
 
+	private boolean _relatedObjectHasBeenHandled(
```

Comment:
We should rename, from Object to ObjectDefinition:

`_isObjectDefinitionHandled`